rflink/parser.py

```diff
@@ -6,7 +6,29 @@
 from typing import Any, Callable, Dict, Generator, cast
 
 UNKNOWN = 'unknown'
-SWITCH_COMMAND_TEMPLATE = '{node};{protocol};{id};{switch};{command};'
+
+"""
+This template is only compatible with packet structure PACKET_COMMAND (10;NewKaku;0cac142;3;ON;). Since this template is used for encoding the string to the RFLink, this breaks sending Milight commands.
+Instead, this template should support items such as 20;01;MiLightv1;ID=F746;SWITCH=00;RGBW=3c00;CMD=ON; (meaning RGBW value BEFORE command).
+
+# 10;NewKaku;0cac142;3;ON;
+PACKET_COMMAND = DELIM.join(['10', PROTOCOL, ADDRESS, BUTTON, COMMAND])
+# 10;MiLightv1;F746;00;3c00;ON;
+PACKET_COMMAND2 = DELIM.join(['10', PROTOCOL, ADDRESS, BUTTON, VALUE, COMMAND])
+# 10;MERTIK;64;UP;
+PACKET_COMMAND3 = DELIM.join(['10', PROTOCOL, ADDRESS, COMMAND])
+# 10;DELTRONIC;001c33;
+PACKET_COMMAND4 = DELIM.join(['10', PROTOCOL, ADDRESS])
+
+We will create 4 switch command templates, in line with the 4 packet command switches.
+
+
+"""
+SWITCH_COMMAND_TEMPLATE_SWITCH_COMMAND = '{node};{protocol};{id};{switch};{command};'
```

Comment:
Good documentation, providing examples to bring context to the code. Maybe instead of 4 constants making this into a lookup dict like suggested below?